code/modules/surgery/surgery_helpers.dm

```diff
@@ -130,6 +130,16 @@
 	else
 		return 0.5
 
+/proc/get_internals_modifier(mob/M)

```

Comment:
If you really want to go the path of forcing *anesthetic tank only* for this, this proc really should be more descriptive on what it's doing, as get_internals_modifier doesn't say what said modifier is *for*.

Reply:
Done, thank you!